### PR TITLE
Add Plural as user and managed offering

### PR DESCRIPTION
**Pull request by mosabua**
Based on info supplied by @avaidyanatha .. waiting for his review and approval for merge.

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -331,6 +331,18 @@
   logo: /assets/images/logos/quintoandar.png
   logosmall: /assets/images/logos/quintoandar-small.png
   #   https://medium.com/quintoandar-tech-blog/building-a-sql-engine-infrastructure-at-quintoandar-73540e136c4e
+- name: Plural
+  anchor: plural
+  logo: /assets/images/logos/plural.png
```
Comment (mosabua) [inline _data/users.yml:336]: I created these images off your logo on GitHub ..

File: _data/users.yml [hunk 2]
```diff
@@ -331,6 +331,18 @@
...
+  logosmall: /assets/images/logos/plural-small.png
+  testimonial: |
+    Plural creates the future of open-source infrastructure. We are building a
+    flexible, scalable solution to application delivery that gives you the
+    autonomy you need to build and reconfigure the software you use freely. Our
+    offering includes support for Trino.
```
Comment (mosabua) [inline _data/users.yml:342]: I added this sentence.

File: resources.md [hunk 3]
```diff
@@ -242,6 +242,22 @@ versioning.
 
 </div>
 
+<div markdown="1" class="item">
+
+#### [Plural](https://www.plural.sh/)
+
+Plural is an open-source, self-hosting framework that allows you to deploy Trino
```
Comment (mosabua) [inline resources.md:249]: This is the text supplied by you

**Review by avaidyanatha — APPROVED** [review]
this looks great, thanks @mosabua!

**Pull request merged**